api-search/src/main/groovy/org/cedar/onestop/api/search/service/LogstashElasticService.groovy

```diff
@@ -0,0 +1,111 @@
+package org.cedar.onestop.api.search.service
+
+import groovy.json.JsonOutput
+import groovy.json.JsonSlurper
+import groovy.util.logging.Slf4j
+import org.apache.http.entity.ContentType
+import org.apache.http.nio.entity.NStringEntity
+import org.elasticsearch.client.Response
+import org.elasticsearch.client.RestClient
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+
+@Slf4j
+@Service
+class LogstashElasticService {
```

Comment:
The two logstash service classes suggest we need to update our package structure for more clarity (and rename ElasticsearchService, since that is no longer a generic thing). We will need to take into account some of the new parts of the sitemap branch, but we have several classes between the controller and service packages that are really just helper classes.